pages/common/yesod.md

```diff
@@ -1,9 +1,9 @@
 # yesod
 
-> A framework to develop Haskell web applications.
-> All yesod commands are invoked through the `stack` project manager.
+> Helper tool for the Yesod Haskell-based web framework.
```

Comment:
Somehow the "Yeson haskell-based" feels odd when I say it. I don't know how else to put it though. I have come up with "Helper tool for Yesod; a haskell-based web framework", but I think its can be improved.